sdcm/utils/cloud_monitor/templates/per_qa_user.html

```diff
@@ -1,4 +1,4 @@
-    <h2>Instances</h2>
+    <h2>Instances are running</h2>
```

Comment:
`Running instances` or `Instances that are running`

Reply:
fixed